tests/unit/plugins/module_utils/modules/ansible_aws_module/test_minimal_versions.py

```diff
@@ -175,9 +175,10 @@ def test_warn_boto3_and_botocore(self, monkeypatch, stdin, capfd):
 
         warning_dict = dict()
         for warning in warnings:
+            warning = str(warning)
             if "boto3" in warning:
                 warning_dict["boto3"] = warning
-            if "botocore" in warning:
+            if "botocore" in str(warning):
```

Comment:
```suggestion
            if "botocore" in warning:
```

warning has already been stringified just above